app/mailers/subscription_mailer.rb

```diff
@@ -0,0 +1,9 @@
+class SubscriptionMailer < ApplicationMailer
+  default from: 'subscriptions@netflix.com'
+
+  def expiring(subscription)
+    @subscription = subscription
+    mail to: subscription.users.pluck(:email), subject: 'Netflix Subscription Expiring Soon'
```

Comment:
i only want to send email to enabled users

Reply:
Association added for enabled_users.